prestoadmin/__init__.py

```diff
@@ -20,7 +20,7 @@
 
 from fabric.api import env
 
-__version__ = '1.4-SNAPSHOT'  # Make sure to update setup.py too
+__version__ = '1.4'
```

Comment:
I couldn't find where this was referencing in setup.py. I assumed it was outdated..
What specifically do I need to update?